### ISSUE TITLE
Observation/Question - new ships in blueprint menu

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#4392]
As of this latest build (21w51b) the blueprint menu contains three ships which are not mine and also not from Steam. They filter under the local category.

Are these ships meant to be there for us to play with?

If so, can I humbly request that we have a third filter category for in-game ships.

**Comment by tsunamayo** [comment]
@cloudrambler yep, I also wanted to add a filter as I  will add more default blueprints. It was for newcomers, so they can check some ship out of the box. Thanks

**Comment by cloudrambler** [comment]
That's cool. I vote for the Rocket symbol to denote the third filter category.

**Comment by tsunamayo** [comment]
@cloudrambler yeah actually the look of the icon was my nb1 issue ;)
